app.js

```diff
@@ -1,13 +1,23 @@
 const express = require('express')
 
-const {
-    getCategories
-} = require("./controllers/categories");
+const { getCategories } = require("./controllers/categories");
+const { getReview } = require("./controllers/review")
 
 const app = express()
 
+app.use(express.json());
+
 app.get('/api/categories', getCategories);
 
+app.get('/api/reviews/:review_id', getReview)
+
+app.use((err, req, res, next) => {
+    if (err.status === 404) {
+      res.status(404).send({ message: 'not found' })
```

Comment:
You could just say `if(err.status) { res.status(err.status)....}` by using the truthy nature of err.status having a value.